lib/rubocop/cop/packaging/require_with_relative_path.rb

```diff
@@ -64,26 +65,39 @@ def on_new_investigation
         def on_send(node)
           return unless require?(node)
 
-          add_offense(node)
+          add_offense(node) do |corrector|
+            corrector.replace(node, good_require_call)
+          end
+        end
+
+        # Called from on_send, this method helps to replace
+        # the "bad" require call with the "good" one.
+        def good_require_call
+          bad_part = @str.match(%r{.*/lib/}).to_s
+          good_call = @str.delete_prefix(bad_part)
```

Comment:
I would replace these two lines by

```ruby
good_call = @str.sub(%r{^.*/lib/}, "")
```

Reply:
This is great! Shorter and much simpler, thanks!
I also missed a `^`, which was failing in cases like `/lib/foo/bar`.

Added a commit in your name, thanks again! 🚀